ts/Series/PointAndFigure/PointAndFigureSeriesDefaults.ts

```diff
@@ -0,0 +1,147 @@
+/* *
+ *
+ *  (c) 2010-2024 Kamil Musialowski
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type PointAndFigureSeriesOptions from './PointAndFigureSeriesOptions';
+
+/* *
+ *
+ *  API Options
+ *
+ * */
+
+/**
+ * The Point and Figure series represents changes in stock price movements,
+ * without focusing on the time and volume. Each data point is created when the
+ * `boxSize` criteria is met. Opposite column of points gets created only when
+ * the `reversalAmount` threshold is met.
+ *
+ * @sample stock/demo/pointandfigure/
+ *         Point and Figure series
+ *
+ * @extends      plotOptions.scatter
+ * @product      highstock
+ * @requires     modules/pointandfigure
+ * @optionparent plotOptions.pointandfigure
+ */
+
+const PointAndFigureSeriesDefaults: PointAndFigureSeriesOptions = {
+    boxSize: '1%',
+    reversalAmount: 3,
+    tooltip: {
+        pointFormat: '<span style="color:{point.color}">\u25CF</span> ' +
+          '<b> {series.name}</b><br/>' +
+          'Close: {point.y:.2f}<br/>',
+        headerFormat: ''
+    },
+    turboThreshold: 0,
+    groupPadding: 0.2,
+    pointPadding: 0.1,
+    pointRange: null,
+    dataGrouping: {
+        enabled: false
+    },
```

Comment:
there is a property on series `allowDG`. set that instead of setting the default value here. It might break if user overwrites options through API

Reply:
Thanks!